### PR TITLE
Switch wiki deployment to the orabot user with SSH auth

### PR DESCRIPTION
**Pull request by obrakmann**
I'm not really sure if this is enough to fix the deployment, since IIRC the "Updating https://github.com/..." line didn't appear in the travis build log, so the script must have either not run at all or bailed out before even doing anything git-related. Anyhow, that's why I added another echo at the start of the script to see what's going on.

Once this is merged, this repo needs to use orabot's creds for the travis-ci service.

### CODE REVIEW

**Comment by Mailaender** [comment]
> Once this is merged, this repo needs to use orabot's creds for the travis-ci service.

Already updated https://github.com/OpenRA/OpenRA/settings/hooks/772373.

File: packaging/update-wiki.sh [hunk 1]
```diff
@@ -27,6 +36,8 @@ git config --local user.email "orabot@users.noreply.github.com"
 git config --local user.name "orabot"
 git add "Traits${TAG}.md"
 git add "Lua API${TAG}.md"
-git commit -m "Update trait and scripting documentation for branch $1"
+git commit -m "Update trait and scripting documentation for branch $1" &&
```
Comment (Mailaender) [inline packaging/update-wiki.sh]: Why is this `&&` necessary?

Comment (obrakmann) [inline packaging/update-wiki.sh]: `git commit` returns 1 when there was nothing to commit, in which case we can skip the push

**Comment by Mailaender** [comment]
:+1: There is no other way to test it than merge and watch it during the next tag creation.

**Pull request merged**